wright/src/grammar/parsers/expression/block.rs

```diff
@@ -1,12 +1,41 @@
-use crate::grammar::ast::{eq::AstEq, Block, Expression};
+use crate::grammar::ast::{eq::AstEq, Block, Expression, Statement};
 use crate::grammar::model::{Fragment, HasFragment};
 use crate::grammar::parsers::expression::ToExpression;
+use crate::grammar::parsers::whitespace::token_delimiter;
+use crate::grammar::parsers::with_input;
+use nom::bytes::complete::tag;
+use nom::combinator::{map, opt};
+use nom::multi::many0;
+use nom::sequence::{delimited, pair, terminated};
 use nom::IResult;
 
 impl<'s> Block<'s> {
+    /// Block delimiters. Should not change.
+    pub const DELIMITERS: (&'static str, &'static str) = ("{", "}");
```

Comment:
Consider making this a `[&'static str; 2]`

Reply:
Actually `DELIMITER_LEFT` and `DELIMITER_RIGHT` might be better

Reply:
I'll refactor with `Start_delimiter` and `end_delimiter`